mozilla/foundation.mozilla.org
# Collection fix via data migration

Closes #6720 
Related PRs/issues #6702 

**Note**: This migration _needs_ to be run in production before #6702 can be applied. #6702 will fail if the watail migrations are run before the migration in this PR. 

Steps to test: 
1. Checkout this branch
2. Pull down the staging database
3. Run `inv manage clean_collections` 

All migrations should run. And if you go to http://localhost:8000/cms/collections/ you should see our collections show up the way they're supposed to. 

Let's also double check that the Collections that end up being deleted in the 0008 migration are the in fact factory collections, and not actually being used. As far as I was able to tell today, this is wiping out unused collections from staging's database.
